# file: C:\Users\erena\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pandas\errors\__init__.py
# hypothesis_version: 6.140.2

['AbstractMethodError', 'CSSWarning', 'ClosedFileError', 'DataError', 'DatabaseError', 'DtypeWarning', 'DuplicateLabelError', 'EmptyDataError', 'IndexingError', 'IntCastingNaNError', 'InvalidColumnName', 'InvalidComparison', 'InvalidIndexError', 'InvalidVersion', 'LossySetitemError', 'MergeError', 'NoBufferPresent', 'NullFrequencyError', 'NumbaUtilError', 'OptionError', 'OutOfBoundsDatetime', 'OutOfBoundsTimedelta', 'ParserError', 'ParserWarning', 'PerformanceWarning', 'PyperclipException', 'SettingWithCopyError', 'SpecificationError', 'UnsortedIndexError', '_cacher', '_item_cache', 'classmethod', 'method', 'property', 'staticmethod']